docs/configuration/overview.mdx

```diff
@@ -364,4 +364,21 @@ Whether to include commit bodies in the changelog. Defaults to `false`.
 
 #### onlyBreaking
 
-Whether to include only breaking changes in the changelog. Defaults to `true`.
\ No newline at end of file
+Whether to include only breaking changes in the changelog. Defaults to `true`.
+
+### changelogFormat
+
+Configure the format of the generated CHANGELOG.md.
+
+```yaml
+command:
+  version:
+    changelogFormat:
+      includeDate: true
+```
+
+#### includeDate
+
+Whether to include the date in the generated CHANGELOG.md. Defaults to `false`.
+
+With enabled, changelog entry header will include the date in the `yyyy-MM-dd` format.
```

Comment:
It should have an empty line at the end of the markdown file (we really should add a markdown linter to the pipeline)